src/carousel/components/iframe-authoring.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { ChangeEvent, useEffect, useRef, useState, useCallback } from "react";
+import { FieldProps } from "react-jsonschema-form";
+import { IframePhone } from "../../shared/types";
+import iframePhone from "iframe-phone";
+import deepEqual from "deep-equal";
+
+import css from "./iframe-authoring.scss";
+import { v4 as uuidv4 } from "uuid";
+
+// This is only temporary list. In the future, it will be replaced by LARA Interactive API call that returns all the available managed interactives.
+const carouselQuestionSegment = /carousel\/?$/;
+const availableInteractives = [
+  {
+    url: "",
+    name: "Select an interactive"
+  },
+  {
+    url: window.location.href.replace(carouselQuestionSegment, "open-response"),
+    name: "Open response"
+  },
+  {
+    url: window.location.href.replace(carouselQuestionSegment, "fill-in-the-blank"),
+    name: "Fill in the blank"
+  },
+  {
+    url: window.location.href.replace(carouselQuestionSegment, "multiple-choice"),
+    name: "Multiple choice"
+  },
+  {
+    url: window.location.href.replace(carouselQuestionSegment, "video-player"),
+    name: "Video"
+  }
+];
```

Comment:
This will cause the same bug as Scaffolded Question one: https://www.pivotaltracker.com/story/show/175502689
But it's fine to leave it like that. I'll update SQ soon, so then we can copy the same approach here.